ee/fe/modules/340-extended-monitoring/docs/CONFIGURATION.md

```diff
@@ -12,6 +12,7 @@ Attach the `extended-monitoring.flant.com/enabled` annotation to the Namespace t
 - adding the appropriate helm-chart to the project (recommended method);
 - adding it to `.gitlab-ci.yml` (kubectl patch/create);
 - attaching it manually (`kubectl annotate namespace my-app-production extended-monitoring.flant.com/enabled=""`).
+- configuring via [namespace-configurator](https://deckhouse.io/en/documentation/v1/modules/600-namespace-configurator/) module.
```

Comment:
@z9r5, could you help with the relative links here?

Reply:
I was considering to use relative links, but found out that all the documentation is using absolute ones

https://github.com/deckhouse/deckhouse/search?q=https%3A%2F%2Fdeckhouse.io%2F

Reply:
fixed

Reply:
I'm not sure about the right way to provide cross-links. 

We have two versions of our docs-webserver. The one is deployed to the flat internal cluster and can be reached by the deckhouse.io domain. The other one is deployed to the client cluster (as one of the Deckhouse modules) and can be reached by the public domain. 

I thought that it may be wrong to give deckhouse.io links from in-cluster docs.

Reply:
Agree, I prepare a new issue to fix that.

Reply:
BTW, just checked that, the shipped version of the documentation has all links replaces to the correct ones.

Reply:
done https://github.com/deckhouse/deckhouse/issues/484